tasks/src/unconditional-image-generation/data.ts

```diff
@@ -0,0 +1,55 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "NFT images dataset of 'cryptopunks' for unconditional image generation.",
```

Comment:
It would be nice to have a generic dataset like CIFAR or CelebA. https://huggingface.co/datasets/cifar100 this is more of a general purpose dataset that would be useful for use cases 🙂

Reply:
Sure! I am adding CIFAR-100 and CelebA as datasets here.